PluralKit.Core/Database/Migrations/12.sql

```diff
@@ -0,0 +1,10 @@
+-- SCHEMA VERSION 12: <insert date> --
+-- Changelog:
+-- * Add disabling front/latch autoproxy per-member
+-- * Add configurable latch timeout duration
+-- * Add disabling autoproxy per-account
+
+alter table members add column disable_autoproxy bool not null default false;
+alter table systems add column latch_timeout int not null default 6;
```

Comment:
Perhaps leave this null by default and have a fallback defined in code? That way that can be changed separately

Reply:
I updated this in the new PR.